File: Leetcode String/2186. Minimum Number of Steps to Make Two Strings Anagram II.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        int count=0;
        unordered_map<char,int> m,m1;
        for(auto i:s)
            m[i]++;
        for(auto i:t)
            m1[i]++;
        for(int i=0; i<26; i++){
            if(abs(m[i + 'a'] - m1[i+'a']) != 0)
                count = count + abs(m[i + 'a'] - m1[i+'a']);
        }
        return count;
    }
};

File: java/leetcode java/2186. Minimum Number of Steps to Make Two Strings Anagram II.java
import java.util.*;
class Solution {
    public int minSteps(String s, String t) {
        Map<Character,Integer> m = new HashMap<>();
        Map<Character,Integer> m1 = new HashMap<>();
        for(int i=0;i<s.length();i++){
            char c= s.charAt(i);
            m.put(c,m.getOrDefault(c,0)+1);
        }
        int cnt=0;
        for(int i=0;i<t.length();i++){
            char c=t.charAt(i);
                m1.put(c,m1.getOrDefault(c,0)+1);
        }
        for(int i=0;i<26; i++){
            if(Math.abs(m.getOrDefault(((char)(i+'a')),0) - m1.getOrDefault(((char)(i+'a')),0))  != 0)
                cnt+= Math.abs(m.getOrDefault(((char)(i+'a')),0) - m1.getOrDefault(((char)(i+'a')),0));
        }
        return cnt;
    }
}